src/pzt/mysql/sections/20-operationalize/20-deployments/kubernetes/10-certificate.mdx

```diff
@@ -0,0 +1,70 @@
+<p>
+  There are various approaches for configuring TLS certificate renewal in
+  Kubernetes. For this tutorial, we will assume you are using{' '}
+  <a href="https://helm.sh/">Helm</a> and will manage certificate renewal using
+  the open source <a href="https://cert-manager.io/docs/">cert-manager</a>{' '}
+  project. You may repurpose various pieces for your own configuration by
+  following the cert-manager or {props.pzt.name} documentation.
+</p>
+
+First, install cert-manager in your cluster with Helm:
+
+```shell
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --set installCRDs=true
+```
+
+In cert-manager, a custom resource type `Certificate` represents
+each TLS certififcate and specifies how to store it.
+
+<p>
+  Create a new <inlineCode>Certificate</inlineCode> resource{' '}
+  <inlineCode>{props.pzt.server.name}-certificate.yaml</inlineCode> for your{' '}
+  {props.pzt.name} server. Make sure you specify the same DNS name configured
+  above, and keep track of the Kubernetes secret name where cert-manager will
+  store your certificate and private key.
+</p>
+
+<CodeBlock autoCopy language="yaml">
+  {`apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: ${props.pzt.server.name}
+spec:
+  secretName: ${props.pzt.server.name}-tls
+  duration: 24h
+  keySize: 2048
+  keyAlgorithm: rsa
+  keyEncoding: pkcs1
+  dnsNames:
+    - ${props.pzt.server.dnsName}
+  usages:
+    - digital signature
+	- key encipherment
+	- server auth
+  issuerRef:
+    name: my-ca
+    kind: ${props.page.provisioner === 'jwk' ? 'StepIssuer' : 'Issuer'}
+    group: ${
+      props.page.provisioner === 'jwk'
+        ? 'certmanager.step.sm'
+        : 'cert-manager.io'
+    }`}
+</CodeBlock>
+
+Customize the `Certificate` name, `secretName`, and issuer name to suit your own naming style in your Kubernetes cluster, then apply it with `kubectl`.
+
+<CodeBlock autoCopy language="shell">
+  {`kubectl apply -f ${props.pzt.server.name}-certificate.yaml`}
+</CodeBlock>
+
+<p>
+  This configuration specifies that cert-manager should issue and renew a TLS
+  certificate with the DNS name{' '}
+  <inlineCode>{props.pzt.server.dnsName}</inlineCode> and store the certificate
+  and private key in a Kubernetes secret named{' '}
+  <inlineCode>{props.pzt.server.name}-tls</inlineCode>. The certificate is valid
+  for 24 hours, and cert-manager will automatically renew it before expiration
+  and update the <inlineCode>{props.pzt.server.name}-tls</inlineCode> secret.
+</p>
```

Comment:
What's the difference from this one and the default? Just the `Certificate` resource configuration? Maybe we should create a snippet for that (eg. `snippets/certificate-resource`) so there's less duplication?